models/JobModel.js

```diff
@@ -13,6 +26,10 @@ const JobSchema = new Schema(
     locations: [String],
     vacancies: { type: Number, require: true },
     isDeleted: { type: Boolean, default: false },
+    applicants: {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 4 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7787263)
```suggestion
		applicants: {
```